.env

```diff
@@ -39,3 +37,13 @@ STOP_HELING_BASE_URL: something
 STOP_HELING_API_KEY: something
 STOP_HELING_APP_ID: something
 VERLOREN_OF_GEVONDEN_BASE_URL: something
+STOLEN_ALERT_TWITTER_1_USERNAME: test
```

Comment:
Is there a better way to store these? This seems sort of gross.

Reply:
Can you elaborate? I'm not sure what gross is calling out here, so I don't know what would constitutes "better" conditional on that metric. 

If it's something to do with the naming pattern, that's an attempted improvement over the following original naming:

```
TEST_SCREEN_NAME
CONSUMER_KEY
CONSUMER_SECRET
ACCESS_TOKEN
ACCESS_TOKEN_SECRET
OMNIAUTH_CONSUMER_KEY
OMNIAUTH_CONSUMER_SECRET
SECOND_TEST_SCREEN_NAME
SECOND_ACCESS_TOKEN
SECOND_ACCESS_TOKEN_SECRET
```

Reply:
If feels like there are a bunch of duplicates, and if there was a way to pass them more directly in, that would be cool. But I understand why that would be much more difficult.

I don't have a good answer to what to do instead. You can ignore this.

Reply:
These env vars are only needed in `.env.test`—they're only in `.env` here for documentation. I'll add a note to the README. Happy to remove them from `.env` if you think their inclusion creates confusion.

Reply:
Nah. It’s fine. Skip it